### ISSUE TITLE
Are we compatibable with mmcv 1.6.0 now?

### ISSUE THREAD

**Issue by makecent** [open-mmlab/mmaction2#1738]
I use `mim install mmaction2` and got an error:
```shell
  File "/home/louis/miniconda3/envs/open-mmlab/lib/python3.9/site-packages/mmaction/__init__.py", line 11, in <module>                                                                                     
    assert (digit_version(mmcv_minimum_version) <= mmcv_version  
AssertionError: MMCV==1.6.0 is used but incompatible. Please install mmcv>=1.3.6, <=1.5.0.  
```

I have to manually modify the `mmaction/__init__.py` file, and it seems NO incompatibility problem with mmcv=1.6.0. Should we update the `__init__.py` file now?

**Comment by kennymckormick** [comment]
Yes, you can just update the `__init__.py`, guess nothing changed. We will also create a PR to update on our side.

**Comment by kennymckormick** [comment]
@ly015  Please consider to draft a new release to keep the `pip install` up to date.

**Comment by hukkai** [comment]
Just found the maximum version is 1.6.0
https://github.com/open-mmlab/mmaction2/blob/40643bce66e78fbe525c1922329e82480f2aae0b/mmaction/__init__.py#L8

**Comment by makecent** [comment]
However, no matter install mmaction2 with:
```bash
mim install mmaction2
```
or
```bash
mim install mmaction2 -f https://github.com/open-mmlab/mmaction2.git
```
The `maximum_version` in `__init__.py` of `mmaction2` is still `1.5.0`

**Comment by kennymckormick** [comment]
@hukkai Yes, I mean `pip install` still uses an old version of this codebase. And the most easy solution is to draft a new release.

**Comment by ly015** [comment]
This issue has been fixed by 699e31a05ccad111c2b0da31c31323fc6e5097a2
